contents/blog/raising-3m-for-os.md

```diff
@@ -0,0 +1,230 @@
+---
+date: 2020-06-02
+title: How we raised $3M
```

Comment:
As a meta point on the entire article, I wasn't sure who the article was written for. I think it's worth having a single person in mind, and having a title like "How to raise $3m for an open source project" so that the article feels a bit more coherent.